tsconfig.json

```diff
@@ -8,6 +8,13 @@
      * due to import of types from typechain-types.
     */
     "importsNotUsedAsValues": "remove",
+    /*
+     * Adding `undefined` to array item type makes less sense 
+     * in testing, where items of arrays in runtime are 
+     * generally guaranteed to exist, thus we overwrite
+     * `noUncheckedIndexedAccess` to false.
+     */
+    "noUncheckedIndexedAccess": false,
```

Comment:
maybe we need to set this back to true in the `tsconfig.build.json`

Reply:
@rsksmart/payments what do you think?

Reply:
I think that its ok, but lets see if someone else has something to say.

Reply:
Instead of disabling this globally, I'd have preferred to force the type where necessary

Reply:
Sure, however that requires changes in 14 files as this causes 
<img width="1043" alt="a" src="https://user-images.githubusercontent.com/36888576/218429476-6f4f3191-612e-4f40-a198-a69f465d3626.png">

Reply:
As discussed, we leave it set to false for now, as it would require changes in 14 files. Should we create task for this @antomor ?

Reply:
Done, https://rsklabs.atlassian.net/browse/PP-678